elastic/kibana
# [APM] ADD logic to capture telemetry for metricset with and without rollups

## Summary

Resolves 
1. https://github.com/elastic/kibana/issues/161981
2. https://github.com/elastic/kibana/issues/161984
3. It also adds the same statistics for last day

## How to test

A deployment has been prepared using the Chrome Extension. (It uses Obs Environment, so data is present)

Please go to this end point to see a similar payload in screenshot

> https://achyutjhunjhunwala-d-pr166598.kb.us-west2.gcp.elastic-cloud.com/internal/apm/debug-telemetry

<img width="667" alt="image" src="https://github.com/elastic/kibana/assets/7416358/811b0394-02f1-4edc-aec0-4bb6a3cd14ae">
